app/controllers/concerns/application_controller/authorization.rb

```diff
@@ -43,4 +43,15 @@ def handle_access_denied(exception)
       redirect_to(unauthorized_path)
     end
   end
-end
\ No newline at end of file
+
+  def require_recent_login(options={})
+    unless current_user && current_user.current_login_recent?(options[:max_age])
+      raise RecentLoginRequiredError
+    end
+  end
```

Comment:
Also curious why you did it this way (raising an error) instead of just calling handle_recent_login_required directly. I'm sure there is a reason, just curious what it is...